docs/Hive/Setup.md

```diff
@@ -0,0 +1,87 @@
+# Hive Setup Guide
+
+Follow this setup guide to create your own instance of the Hive back-end.
+
+## Running Hive With Docker
+
+If you are accustomed to using and operating Docker containers, running Hive through Docker may be appealing to you.
+
+Included in the Hive repository is the [`docker-compose.yml`](https://github.com/Atlas-Rhythm/Hive/blob/master/docker-compose.yml) file. To run Hive in a Docker environment, please download this file.
+
+**Do not use this file as-is!**
+
+The Docker Compose file contains some configuration that should be changed before deployment. Please open `docker-compose.yml` in any text environment, and make any necessary modification and configuration to match your deployment environment.
+
+When you are ready to deploy Hive, run `docker-compose up`.
+
+## Downloading Hive
+
+If you do not want to use Docker, you can download the latest release of Hive through [the Releases page](https://github.com/Atlas-Rhythm/Hive/releases).
+
+Simply unzip the Hive release wherever you wish to host Hive.
```

Comment:
Do you also need plugins for this? Or do the releases have plugins bundled with them?

Reply:
If not, might be nice to clarify

Reply:
I think ideally, when we do go to make releases, we'd want to bundle HiveCorePlugins into the release zip. A major pain point in deployment right now is having to manually download and install the HiveCorePlugins to even use Hive, so I believe bundling core plugins wherever possible is a good idea.